core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -1262,6 +1315,21 @@ private void validNameOrThrow(CharSequence tableName) {
         }
     }
 
+    private TableMetadata validateDesiredMetadataVersion(TableToken tableToken, TableMetadata metadata, long desiredVersion) {
+        if (desiredVersion != TableUtils.ANY_TABLE_VERSION && metadata.getMetadataVersion() != desiredVersion) {
+            final TableReferenceOutOfDateException ex = TableReferenceOutOfDateException.of(
+                    tableToken,
+                    metadata.getTableId(),
```

Comment:
```suggestion
                    tableToken.getTableId(),
```

Reply:
fixed in https://github.com/questdb/questdb/pull/4044/commits/1b26f6db43a998a89649ecb8f103a9fc2740ad43